automation/vars/uni-delta.yaml

```diff
@@ -0,0 +1,55 @@
+---
+vas:
+  uni-delta:
+    stages:
+      - path: examples/dt/uni-delta/control-plane/nncp
+        validations:
+          - >-
+            oc -n openstack wait nncp
+            -l osp/nncm-config-type=standard
+            --for jsonpath='{.status.conditions[0].reason}'=SuccessfullyConfigured
+        values:
+          - name: network-values
+            src_file: values.yaml
+        build_output: nncp.yaml
+
+      - path: examples/dt/uni-delta/control-plane
+        validations:
+          - >-
+            oc -n openstack wait osctlplane controlplane
+            --for condition=Ready
+            --timeout=600s
+        values:
+          - name: network-values
+        build_output: control-plane.yaml
+
+      - path: examples/dt/uni-delta/ceph
+        validations:
+          - >-
+            oc -n openstack wait osdpd
+            edpm-ceph
+            --for condition=Ready
+            --timeout=1200s
+        values:
+          - name: edpm-ceph-values
```

Comment:
I think this maps to specific datatype in the `ci_gen_kustomize` role and it takes only two values edpm-values and network-values, based on which the CRs are kustomized 
https://github.com/openstack-k8s-operators/ci-framework/blob/main/roles/ci_gen_kustomize_values/tasks/generate_snippets.yml#L22-L47